doc/src/ForHosts.dox

```diff
@@ -76,8 +76,9 @@
  *   @fqref{hostApi.HostInterface.identifier} "identifier" and
  *   @fqref{hostApi.HostInterface.displayName} "displayName".
  *
- * - Create a @fqref{hostApi.ManagerFactory} to bootstrap the API,
- *   provide the following objects:
+ * - Use @fqref{hostApi.ManagerFactory.defaultManagerForInterface}
+ *   "ManagerFactory.defaultManagerForInterface" to bootstrap the
+ *   API, provide the following objects:
```

Comment:
[question] Does this need a note that you also need a toml config file?  Or are we assuming some sysadmin has set that up prior, for the purposes of these instructions?

Reply:
Added some further notes [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/608/commits/16df21f7d728738136b1f8f4ac34e8ad00d64816), helpful?